prime-angular-frontend/src/app/config/config.model.ts

```diff
@@ -66,7 +66,7 @@ export interface CollegePracticeConfig {
   practiceCode: number;
 }
 
-export interface VendorConfig extends Config<number> {
+export class VendorConfig extends Config<number> {
```

Comment:
Does this need to be a `class`?

Reply:
Forgot why I changed it. but I reversed it back to interface.